dsl_parser/elements/node_templates.py

```diff
@@ -149,7 +149,7 @@ def parse(self, relationships, data_types):
 class NodeTemplateInstancesDeploy(Element):
 
     required = True
-    schema = Leaf(type=int)
+    schema = Leaf(type=(int, dict))
 
     def validate(self):
         if self.initial_value < 0:
```

Comment:
if it's dict, how does this line make sense?

i'd improve the validation so that if its a dict i'd try to parse it via intrinsic function and thereby make sure the value is valid